src/Core/tests/UnitTests/ImageSource/ImageSourceServiceTests.cs

```diff
@@ -25,6 +80,52 @@ public void ResolvesToConcreteTypeOverInterface()
 			Assert.IsType<UriImageSourceService>(service);
 		}
 
+		[Fact]
+		public void GetsImageSourceTypeEvenWhenNoServiceWasRegistered()
```

Comment:
If I'm not mistaken, the 4 ImageSource interfaces are already registered, this and for `GetsImageSourceServiceTypeEvenWhenNoServiceWasRegistered` so the test name is a bit misleading (because the service is registered). Perhaps changing it a type that derives from the custom image source type instead?